### PR TITLE
Updating to use new meta service

### PR DESCRIPTION
**Pull request by gregmccoy**
## Summary

Updating endpoints to use new meta service instead of neo4j

## JIRA Issues

PILOT-908

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Refactor or reformatting

## Testing

Are there any new or updated tests to validate the changes?

- [x] Yes
- [ ] No

## Test Directions

Require a local database which the database changes. Create, approve and complete a copy request

### CODE REVIEW

File: app/commons/meta_services/__init__.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+import httpx
+
+from app.config import ConfigClass
+from app.models.base import EAPIResponseCode
+from app.resources.error_handler import APIException
+
+
+def get_node_by_id(entity_id: str) -> dict:
+    print(entity_id)
```
Comment (afmam) [inline app/commons/meta_services/__init__.py:25]: `print` should be removed here

Comment (gregmccoy) [reply app/commons/meta_services/__init__.py]: Done

File: app/commons/meta_services/__init__.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        raise APIException(error_msg=error_msg, status_code=EAPIResponseCode.internal_error.value)
+    return response.json()['result']
+
+
+def get_files_recursive(entity: dict) -> list:
+    query_data = {
+        'container_code': entity['container_code'],
+        'zone': entity['zone'],
+        'recursive': True,
+        'parent_path': entity['parent_path'] + '.' + entity['name']
```
Comment (afmam) [inline app/commons/meta_services/__init__.py:49]: Using f-strings here instead would help with consistency, as you used them across the service (lines 40 and 54 in this script for example)

Comment (gregmccoy) [reply app/commons/meta_services/__init__.py]: Done

**Review by afmam — APPROVED** [review]

**Pull request merged**